### PR TITLE
R20.1 anchor type

### LINKED ISSUE #210: Add surface type selection for Anchors
Need to reproduce the load dialog for Anchors, and allow users to select different surfaces

### PR DESCRIPTION
**Pull request by jake-kirkland-teton**
OK. The face selection popup has been reworked and should be connected correctly now. If there are any issues, let me know and I will look into them. 

Closes #210

### CODE REVIEW

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml [hunk 1]
```diff
@@ -427,13 +479,21 @@ Item {
...
+                                        else if (selectLoadButton.checked) {
+                                            bcListForces.model.surfaceType = 3;
+                                            loadColumn.iconsEnabled();
+                                        }
                                     }
                                 }
                             }
 
                             Row {
+                                visible: (selectLoadButton.checked) ? true : false
```
[the remark this answers is not in the document]
Comment (bradymadams) [reply SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:495]: The ternary shouldn't even be necessary. `visible: selectLoadButton.checked` should be equivalent and less verbose.

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml [hunk 2]
```diff
@@ -163,24 +172,32 @@ Item {
...
 
-            applyLoadDialog.handler.setPosition(applyLoadDialog.x, applyLoadDialog.y)
+            faceDialog.handler.setPosition(faceDialog.x, faceDialog.y)
+        }
+
+        function comboboxElements() {
+            if (selectLoadButton.checked == true) {
+                comboDialogType.model = ["Push / Pull"];
+            }
+            else if (selectAnchorButton.checked == true) {
```
Comment (bradymadams) [inline SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:191]: `} else if (condition) {`

No need to compare booleans, `if (selectLoadButton.checked)` is sufficient.

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml (hunk 1 above)
[the remark this answers is not in the document]
Comment (jake-kirkland-teton) [reply SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:495]: Valid points. Just changed it, checking it now.

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml [hunk 3]
```diff
@@ -508,140 +570,154 @@ Item {
...
+                                                horizontalCenter: parent.horizontalCenter
+                                                verticalCenter: parent.verticalCenter
+                                            }
+
+                                        Repeater {
+                                            id: tickmarks
+                                            model: loadHelperData.maxValue / loadHelperData.stepSize -1
+                                            Rectangle {
+                                                function indexHelper(index) {
+                                                    console.log(model)
```
Comment (bradymadams) [inline SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:682]: Leftover debug?

Comment (jake-kirkland-teton) [reply SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:682]: Would assume so. Removed.

File: SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml (hunk 2 above)
Comment (jake-kirkland-teton) [reply SmartSlicePlugin/select_tool/SmartSliceSelectTool.qml:191]: Gotcha. That makes sense. Got em fixed.

**Pull request merged**